ament_tools/verbs/build/cli.py

```diff
@@ -114,13 +114,13 @@ def prepare_arguments(parser, args):
         help='End with a particular package',
     )
     parser.add_argument(
-        '--only-package',
-        '--only',
```

Comment:
Any shorter but unique variation of arguments is always available. That's always why renaming it from `--only-package` to `--only-packages` is backward compatible.